tests/src/test_cases/libs/test_freetype.c

```diff
@@ -29,359 +29,359 @@ static const char * UNIVERSAL_DECLARATION_OF_HUMAN_RIGHTS_JP =
-    {NULL, 4, {4834, 2351}, {4879, 2220}, {0, 0}},
-    {NULL, 1, {5746, 1401}, {0, 0}, {0, 0}},
-    {NULL, 2, {6106, 1401}, {0, 0}, {0, 0}},
-    {NULL, 2, {5763, -213}, {0, 0}, {0, 0}},
-    {NULL, 4, {5684, -446}, {5728, -377}, {0, 0}},
-    {NULL, 4, {5541, -565}, {5641, -516}, {0, 0}},
-    {NULL, 4, {5280, -610}, {5433, -606}, {0, 0}},
-    {NULL, 4, {4891, -614}, {5128, -614}, {0, 0}},
-    {NULL, 4, {4888, -455}, {4899, -541}, {0, 0}},
-    {NULL, 4, {4852, -295}, {4878, -369}, {0, 0}},
-    {NULL, 4, {5138, -303}, {5007, -303}, {0, 0}},
-    {NULL, 4, {5311, -295}, {5269, -303}, {0, 0}},
-    {NULL, 4, {5401, -221}, {5385, -295}, {0, 0}},
-    {NULL, 2, {5746, 1401}, {0, 0}, {0, 0}},
-    {NULL, 1, {6887, 2802}, {0, 0}, {0, 0}},
-    {NULL, 2, {8313, 2802}, {0, 0}, {0, 0}},
-    {NULL, 2, {8250, 2507}, {0, 0}, {0, 0}},
-    {NULL, 2, {6824, 2507}, {0, 0}, {0, 0}},
-    {NULL, 2, {6887, 2802}, {0, 0}, {0, 0}},
-    {NULL, 1, {6549, 1253}, {0, 0}, {0, 0}},
-    {NULL, 2, {7827, 1253}, {0, 0}, {0, 0}},
-    {NULL, 2, {7770, 983}, {0, 0}, {0, 0}},
-    {NULL, 2, {6492, 983}, {0, 0}, {0, 0}},
-    {NULL, 2, {6549, 1253}, {0, 0}, {0, 0}},
-    {NULL, 1, {6445, 762}, {0, 0}, {0, 0}},
-    {NULL, 2, {7731, 762}, {0, 0}, {0, 0}},
-    {NULL, 2, {7674, 492}, {0, 0}, {0, 0}},
-    {NULL, 2, {6388, 492}, {0, 0}, {0, 0}},
-    {NULL, 2, {6445, 762}, {0, 0}, {0, 0}},
-    {NULL, 1, {6340, 270}, {0, 0}, {0, 0}},
-    {NULL, 2, {7512, 270}, {0, 0}, {0, 0}},
-    {NULL, 2, {7455, 0}, {0, 0}, {0, 0}},
-    {NULL, 2, {6283, 0}, {0, 0}, {0, 0}},
-    {NULL, 2, {6340, 270}, {0, 0}, {0, 0}},
-    {NULL, 1, {4496, 303}, {0, 0}, {0, 0}},
-    {NULL, 2, {5692, 303}, {0, 0}, {0, 0}},
-    {NULL, 2, {5630, 8}, {0, 0}, {0, 0}},
-    {NULL, 2, {4434, 8}, {0, 0}, {0, 0}},
-    {NULL, 2, {4496, 303}, {0, 0}, {0, 0}},
-    {NULL, 1, {4611, 844}, {0, 0}, {0, 0}},
-    {NULL, 2, {5807, 844}, {0, 0}, {0, 0}},
-    {NULL, 2, {5746, 557}, {0, 0}, {0, 0}},
-    {NULL, 2, {4550, 557}, {0, 0}, {0, 0}},
-    {NULL, 2, {4611, 844}, {0, 0}, {0, 0}},
-    {NULL, 1, {4715, 2720}, {0, 0}, {0, 0}},
-    {NULL, 2, {6517, 2720}, {0, 0}, {0, 0}},
-    {NULL, 2, {6447, 2392}, {0, 0}, {0, 0}},
-    {NULL, 2, {4645, 2392}, {0, 0}, {0, 0}},
-    {NULL, 2, {4715, 2720}, {0, 0}, {0, 0}},
-    {NULL, 1, {4428, 1950}, {0, 0}, {0, 0}},
-    {NULL, 2, {6394, 1950}, {0, 0}, {0, 0}},
-    {NULL, 2, {6326, 1630}, {0, 0}, {0, 0}},
-    {NULL, 2, {4360, 1630}, {0, 0}, {0, 0}},
-    {NULL, 2, {4428, 1950}, {0, 0}, {0, 0}},
-    {NULL, 1, {5370, 3023}, {0, 0}, {0, 0}},
-    {NULL, 2, {5739, 3105}, {0, 0}, {0, 0}},
-    {NULL, 4, {5825, 2855}, {5790, 2998}, {0, 0}},
-    {NULL, 4, {5872, 2613}, {5860, 2712}, {0, 0}},
-    {NULL, 2, {5479, 2499}, {0, 0}, {0, 0}},
-    {NULL, 4, {5434, 2752}, {5469, 2605}, {0, 0}},
-    {NULL, 4, {5370, 3023}, {5400, 2900}, {0, 0}},
-    {NULL, 0, {0, 0}, {0, 0}, {0, 0}},
+    {NULL, 1, {5888, 6889}, {0, 0}, {0, 0}, {0, 0}},
```

Comment:
You can try to update this value. You can use `TEST_PRINTF` [here](https://github.com/lvgl/lvgl/blob/master/tests/src/test_cases/libs/test_freetype.c#L525) to print out `segments_size` and `data_size` and update them to the `outline_data_U9F98` array.

Reply:
You can use ChatGPT or python script to regenerate code based on the `TEST_PRINTF` log content.